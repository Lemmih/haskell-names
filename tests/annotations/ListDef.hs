module ListDef where

data List a = Nil | Cons a (List a)
