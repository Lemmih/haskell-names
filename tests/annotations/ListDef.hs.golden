List     at 3:29 is the type ListDef.List
